SAT/Z3Interfacing.cpp

```diff
@@ -1546,7 +1547,7 @@ Z3Interfacing::Representation Z3Interfacing::getRepresentation(Term* trm)
         //if constant treat specially
         if(trm->arity() == 0) {
           if(symb->integerConstant()){
-            IntegerConstantType value = symb->integerValue();
+            // IntegerConstantType value = symb->integerValue();
```

Comment:
Is this just dead?

Reply:
Dead, feel free to delete. I was getting a warning and didn't want to kill the author's pattern completely.